bin/node/src/chain_spec.rs

```diff
@@ -72,15 +72,15 @@ pub struct ChainParams {
     pub token_symbol: Option<String>,
 
     #[structopt(long, require_delimiter = true)]
-    account_ids: Option<Vec<String>>,
+    authorities_pkeys: Option<Vec<String>>,
 
     #[structopt(long)]
     n_members: Option<u32>,
 }
 
 impl ChainParams {
     pub fn chain_id(&self) -> &str {
-        match &self.chain_name {
+        match &self.chain_id {
```

Comment:
Can we instead do something like adding `default_value = "a0dnet1"` in `#[structopt(...)]` before `chain_id`?

Reply:
Actually, you can add a task in jira instead

Reply:
done